### ISSUE TITLE
Do complete email verification when importing emails

### ISSUE THREAD

**Issue by karuppiah7890** [freeCodeCamp/mail-for-good#228]
We could use something like [trumail](https://github.com/sdwolfe32/trumail)

**Comment by MitchMo** [comment]
I've been playing around with the Trumail API, and it is telling me, emails from my current work's domain are not deliverable. They are from Office 365. Something like this could get very time consuming as well with each API call

**Comment by karuppiah7890** [comment]
@MitchMo Oh. Well, it was just a thought. It's not like we should necessarily use trumail, anything similar to it. And yes, it is time consuming, but it's worth it if people consider validating the email instead of sending and seeing it get bounced and then manually remove them from list in that case

**Comment by MitchMo** [comment]
No I agree, It would be a nifty feature if you could find the right tool for the job. I like the idea of what you described giving them the option to remove the emails manually once the validation has completed. With that approach you could make something like trumail work

**Comment by karuppiah7890** [comment]
Yup and it's also quite good to verify the validity of the email before
sending even the first mail, after which you will surely know if the mail
exists or not. And yes, it's a quite nifty feature
